### ISSUE TITLE
Please add a brief but descriptive title

### ISSUE THREAD

**Issue by IlCanePerIlGatto** [cabaletta/baritone#2172]
## What do you need help with?
How do I prevent baritone from going to the specific location that a schematic got built? I use impact client. (I want to build a schematic in a specific location)

**Comment by l1ving** [comment]
You want to avoid previously built schematics?

**Comment by IlCanePerIlGatto** [comment]
> You want to avoid previously built schematics?

No, I want to make baritone build schematics in a specific location, not where they got originally built.

**Comment by IlCanePerIlGatto** [comment]
> #build blah.schematic x y z

It doens't work

**Comment by scorbett123** [comment]
what about it isn't working, is there an error in the chat?
